modules/tests/shared/src/test/scala/CommandTest.scala

```diff
@@ -94,6 +94,11 @@ class CommandTest extends SkunkTest {
          WHERE id = $int4
        """.command
 
+  val explainCity: Query[Void, String] =
```

Comment:
Any reason to put this in CommandTest.scala instead of QueryTest.scala?

Reply:
Looking at the content of the files, `QueryTest.scala` seems to be testing the methods on `Query` class, whereas `CommandTest.scala` appears to be testing SQL statements in general (maybe it would be more logical to call it `StatementTest.scala`).
It seems more appropriate to keep it there, but happy to move to `QueryTest.scala` if you prefer; let me know :+1:

Reply:
Moved test into `QueryTest`.